README.md

```diff
@@ -2475,6 +2477,137 @@ It is based on the MBeans ```java.nio:type=BufferPool,name=<pool_name>```.
 Adds a number of metrics related to the class loading system of the JVM.
 It is based on ```java.lang.management.ClassLoadingMXBean```.
 
+## Integration
+
+### Dropwizard MetricRegistry integration
+
+#### Export using DropwizardMetricRegistryDefaultInstanceSamplesProvider
+
+`DropwizardMetricRegistryDefaultInstanceSamplesProvider` helps to export metrics from existing `com.codahale.metrics.MetricRegistry` using [Metrics Reporters](#metrics-reporters) (except PrometheusMetricsExporter).
+
+For example,
+```java
+MetricRegistry registry = new MetricRegistry();
+
+AtomicInteger g1 = new AtomicInteger();
+registry.gauge("g1", () -> new Gauge<Integer>() {
+  @Override
+  public Integer getValue() {
+    return g1.incrementAndGet();
+  }
+});
+
+Counter c1 = registry.counter("c1");
+c1.inc();
+
+Histogram h1 = registry.histogram("h1");
+h1.update(10);
+h1.update(100);
+h1.update(1000);
+
+Timer t1 = registry.timer("t1");
+t1.update(Duration.ofMinutes(10));
+
+Meter m1 = registry.meter("m1");
+m1.mark();
+m1.mark(10);
+
+DropwizardMetricRegistryDefaultInstanceSamplesProvider provider = new DropwizardMetricRegistryDefaultInstanceSamplesProvider(registry);
+
+TelegrafMetricsJsonExporter telegrafMetricsJsonExporter = new TelegrafMetricsJsonExporter(true, provider);
+MetricsJson metrics = telegrafMetricsJsonExporter.exportMetrics();
+
+ObjectMapper mapper = new ObjectMapper();
+System.out.println(mapper.writeValueAsString(metrics));
+```
+Output:
+```text
```

Comment:
I'd suggest using the bash highlighting for consistency with the other parts of the doc.

Reply:
fixed, thank you